### PR TITLE
Add summary API for ContactNDoH cases

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

File: covid_cases/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Summary business logic
+For the summary endpoint, /v2/covidcases/contactndoh, there is some business logic to determine what the latest numbers are from the data in the database.
+
+For the daily image, and the counters on the homepage, we just service the latest that we have in the database.
+
+For the timestamp, we take that from the last update of the WardCase data. This is a tricky one, since the sources could all be updated at different times, but we're only showing one timestamp to the user.
+
+For the "latest" count, we first try to use the counters on the homepage. We look for the latest counter, and then see if there's a counter for the day before that. If there is, then the different is the latest count. Otherwise we go to the WardCase data, where we do a similar thing: we look at the latest date that we have data for, see if there's data for a day before that, and if there is we serve the difference. If there isn't, then we fall back to the "latest" field in the data. The reason we prioritise looking at the total difference first before looking at the latest field, is that the latest field isn't always correct.
```
Comment (erikh360) [inline covid_cases/README.md]: Did you mean difference: "If there is, then the different is the latest count."

File: covid_cases/tasks.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
     SACoronavirusCounter,
     Ward,
     WardCase,
+    WardCaseQuerySet,
```
Comment (erikh360) [inline covid_cases/tasks.py]: I don't see this being used

**Review by erikh360 — APPROVED** [review]
just the two small comments

**Comment by rudigiesler** [comment]
@erikh360 Thanks! Ready for another review :)

**Review by erikh360 — APPROVED** [review]

**Pull request merged**